### PR TITLE
EXAMPLEAND-45 move resource calls to a separate tab

### PR DESCRIPTION
**Pull request by marcin-onegini**

### CODE REVIEW

File: app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+          @Override
+          public void onSuccess(final UserProfile profile) {
+            callImplicitResourceCallToFetchImplicitUserDetails();
+          }
+
+          @Override
+          public void onError(final OneginiImplicitTokenRequestError error) {
+            onImplicitDetailsFetchFailed(error);
+            @OneginiImplicitTokenRequestError.ImplicitTokenRequestErrorType int errorType = error.getErrorType();
+            if (errorType == OneginiDeviceAuthenticationError.DEVICE_DEREGISTERED) {
```
Comment (robertszuba) [inline app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java]: Should be OneginiImplicitTokenRequestError

File: app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+              new DeregistrationUtil(InfoActivity.this).onDeviceDeregistered();
+            } else if (errorType == OneginiDeviceAuthenticationError.USER_DEREGISTERED) {
```
Comment (robertszuba) [inline app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java]: Should be OneginiImplicitTokenRequestError

File: app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+  }
+
+  private void setupUi() {
+    setupActionBar();
+    setupNavigationBar();
+    setupUserDetails();
+  }
+
+  private void setupUserDetails() {
+    final User user = LoginActivity.selectedUser;
```
[the remark this answers is not in the document]
Comment (marcin-onegini) [reply app/src/main/java/com/onegini/mobile/exampleapp/view/activity/InfoActivity.java]: I wish I could, but there's no guarantee the user will be redirected from the login screen. :(

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -152,6 +148,7 @@
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version"/>
 
+        <activity android:name=".view.activity.InfoActivity"></activity>
```
[the remark this answers is not in the document]
Comment (marcin-onegini) [reply app/src/main/AndroidManifest.xml]: fixed

**Pull request merged**